Makefile

```diff
@@ -243,9 +247,15 @@ testpkg: failpoint-enable
 ifeq ("$(pkg)", "")
 	@echo "Require pkg parameter"
 else
+ifeq ("$(case)", "")
```

Comment:
Please don't change Makefile in feature related PR. I think it's better to open a new PR for it.

Reply:
Sure, I will remove it at last.